src/Visualization/Python/Render1D.py

```diff
@@ -0,0 +1,277 @@
+
+
+def print_var_names(files):
+    '''
+    Print all available variables to screen
+    :param args:
+    :return: None
+    '''
+
+    h5files = get_h5_files(files)
+    volfile = h5files[0]["element_data.vol"]
+    obs_id_0 = next(iter(volfile))
+    variables = list(volfile[obs_id_0].keys())
+    variables.remove("connectivity")
+    variables.remove("InertialCoordinates_x")
+    variables.remove("total_extents")
+    variables.remove("grid_names")
+    print("Variables in H5 file:\n{}".format(list(map(str, variables))))
+    for h5_file in h5files:
+        h5_file.close()
+
+
+def get_data(files, var_name):
+    '''
+    Get the data to be plotted
+    :return: the array of data, coords, and time
+    '''
+
+    time = []
+    coords = []
+    data = []
+    h5files = get_h5_files(files)
+    volfiles = [h5file['element_data.vol'] for h5file in h5files]
+    # Get a list of times from the first vol file
+    ids_times = [(obs_id, volfiles[0][obs_id].attrs['observation_value'])
+                 for obs_id in volfiles[0].keys()]
+    ids_times.sort(key=lambda pair: pair[1])
+    for obs_id, local_time in ids_times:
+        local_coords = []
+        local_data = []
+        ordering = None
+        for volfile in volfiles:
+            try:
+                local_data = (local_data + list(volfile[obs_id][var_name]))
+            except KeyError:
+                print("The variable name {} is not a valid variable. "
+                      "Use '--list-vars' to see the list of variable names in "
+                      "the file(s) \n{}".format(var_name,
+                                                list(map(str, h5files))))
+                sys.exit(1)
+            local_coords = (local_coords +
+                            list(volfile[obs_id]['InertialCoordinates_x']))
+        ordering = np.argsort(local_coords)
+        coords.append(np.array(local_coords)[ordering])
+        data.append(np.array(local_data)[ordering])
+        time.append(local_time)
+
+    for h5file in h5files:
+        h5file.close()
+    return time, coords, data
+
+
+def render_single_time(var_name, time_slice, output_prefix, coords, data):
+    '''
+    Renders a single time snap shot
+    :param args: the command line arguments
+    :return: None
+    '''
+    import matplotlib.pyplot as plt
+
+    plt.xlabel("x")
+    plt.ylabel(var_name)
+    plt.title("t = {:.5f}".format(int(time_slice)))
+    plt.plot(coords[int(time_slice)], data[int(time_slice)], 'o')
+    if output_prefix:
+        logging.info("Writing still to file {}.pdf".format(output_prefix))
+        plt.savefig(output_prefix + ".pdf", format='pdf')
+    else:
+        plt.show()
+
+
+def render_animation(var_name, output_prefix, interval, time, coords, data):
+    '''
+    Render an animation of the data
+    :param args: the command line arguments
+    :return: None
+    '''
+    import matplotlib.pyplot as plt
+    import matplotlib.animation as animation
+    line = ()
+    title = None
```

Comment:
Delete these two new lines 206--207. Instead, move the lines 209--224 down, above line 234. Then add the `global line, title` statement to both functions.